### PR TITLE
Suppress GCC6 indentation warnings

### PR DESCRIPTION
**Pull request by Teteros**
I found the source of my hit and miss compile issues. (like #894 and #901)

GCC6 introduces new warnings, which trigger on some older/unported packages.
(Mainly faust packages/audio and Visualisation lib)

This happens because of the compile flags used on them.
-Wall enables all warnings and -Werror treats them as errors
as -Wmisleading-indentation is a new warning in gcc6, _sloppy_ indentation blocks compiling.
See https://gcc.gnu.org/gcc-6/porting_to.html https://gcc.gnu.org/onlinedocs/gcc/Warning-Options.html

This commit suppresses that particular warning.

Indent warnings still appear on visualisation lib, but since -Werror is not set on that makefile, it compires, so I left it be as it's tarball'd.

### CODE REVIEW

**Comment by kmatheussen** [comment]
Thank you, but the warning seems like a good idea, so I don't want to suppress it from radium source files, only from 3rd party code. Can you fix that?

**Comment by Teteros** [comment]
@kmatheussen this should do what you want.
according to the docs -Werror= overrides the default behaviour, so misleading-indentation will be kept as warning.

Likely a good thing actually, I'd imagine it's easier to catch them all that way.

**Comment by kmatheussen** [comment]
Thanks. I rather prefer -Werror on all types of warnings though (there's so much noise in the terminal that you don't always see the warnings), so I changed this when compiling radium files.

File: Makefile.Qt [hunk 1]
```diff
@@ -81,7 +81,7 @@ S7OPTS ?=
 
 WHICHFAUST = $(shell pwd)/bin/packages/faust2/compiler/faust
 # The include files in audio/faudiostream are copied from Faust 0.9.55.
-FAUST = $(WHICHFAUST) -I $(shell pwd)/bin/packages/faust2/architecture  #`which faust` #/usr/bin/faust #~/site/bin/faust
```
Comment (kmatheussen) [inline Makefile.Qt]: Removed this one too. I don't think that would have compiled. This is faust compiler options, not c/c++ compiler options.

**Pull request merged**